### PR TITLE
Update tools

### PR DESCRIPTION
**Pull request by donny-dont**
Update tools

git -> 2.29.2.3
python -> 3.9.1
pip -> 20.3.1
pywin32 -> 300
cmake -> 3.19.1
ninja -> 1.10.2
xampp -> 8.0.0

### CODE REVIEW

**Review by foopoiuyt — CHANGES_REQUESTED** [review]
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.
ERROR: Could not find a version that satisfies the requirement pywin32==300 (from versions: 222, 223, 224, 225, 226, 227, 228)
ERROR: No matching distribution found for pywin32==300
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.
WARNING: Package(s) not found: pywin32

File: tools/Dockerfile [hunk 1]
```diff
@@ -105,7 +105,7 @@ RUN copy-item C:\tools\python\python.exe C:\tools\python\python2.exe
 # Install pywin32 for python 2
 #--------------------------------------------------------------------
 
-ENV PYWIN32_VERSION 228
+ENV PYWIN32_VERSION 300
```
Comment (foopoiuyt) [inline tools/Dockerfile:108]: It seems like this isn't working for python 2.7, maybe they haven't updated the version for 2.7 past 228?

Comment (donny-dont) [reply tools/Dockerfile:108]: Yea I looked at pip and it has the version for 300 being Python 3.x only. Update just that one or keep it all at 228 for consistency?

Comment (foopoiuyt) [reply tools/Dockerfile:108]: Probably just update the one, at this point we should probably consider python 2 and python 3 as being totally separate.

**Comment by donny-dont** [comment]
@foopoiuyt the build got through to msbuild so these changes should be fine.

There's a 8.0.0 release of XAMPP but it uses a different naming scheme so I didn't include it.

**Review by foopoiuyt — APPROVED** [review]

**Pull request merged**